### PR TITLE
UCL

### PR DESCRIPTION
**Pull request by dpshelio**
Adds UCL details and possible preview via Circle-CI

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -122,4 +122,12 @@ workshop_sched_id: "ComputeCanada_Graham_slurm"
 # workshop_host_login: "login.cirrus.ac.uk"
 # workshop_host_location: "EPCC, The University of Edinburgh"
 # workshop_host_prompt: "[yourUsername@cirrus-login0 ~]$ "
-# workshop_host_homedir: "/lustre/home/tc001"
\ No newline at end of file
+# workshop_host_homedir: "/lustre/home/tc001"
+# Example: Myriad cluster at UCL, University College London
+# workshop_host: "Myriad"
+# workshop_host_id: "UCL_Myriad"
```
Comment (mattagape) [inline _config.yml:128]: I don't understand why the UCL parts of this file are commented out, and the University of Waterloo parts left active.

**Review by mattagape — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (dpshelio) [reply _config.yml:128]: That's the default of the lesson, when we merge it we can then activate so that we can contribute this to upstream.

Comment (mattagape) [reply _config.yml:128]: Yes but this is for our own repo. So our version should have us activated, surely? Or are you going to send the PR upstream as well?

Comment (dpshelio) [reply _config.yml:128]: Our version will be activated, but we need to push this upstream too, that's why I'm separating these things so it's easier to handle.

Comment (mattagape) [reply _config.yml:128]: OK

**Pull request merged**